sweep/sweeper.go

```diff
@@ -454,6 +545,73 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch,
 	}
 }
 
+// bucketForFeeReate determines the proper bucket for a fee rate. This is done
+// in order to batch inputs with similar fee rates together.
+func (s *UtxoSweeper) bucketForFeeRate(
+	feeRate lnwallet.SatPerKWeight) (lnwallet.SatPerKWeight, error) {
+
+	step := lnwallet.SatPerKWeight(s.cfg.FeeRateBucketSize)
+	for bucket := s.relayFeeRate + step; bucket <= s.cfg.MaxFeeRate; bucket += step {
+		if feeRate <= bucket {
+			return bucket, nil
+		}
+	}
```

Comment:
This could also be a simple divide with rounding?

Reply:
Good point, done.